go.mod

```diff
@@ -1,6 +1,8 @@
-module github.com/gpsinsight/terraform-provider-fusionauth
+module github.com/fusionauth/terraform-provider-fusionauth
 
-go 1.20
+go 1.21
```

Comment:
i didn't explicitly alter the go version or add the toolchain statement 

I think it was added after `go mod tidy`. Let me know if I should roll this back along with `go.mod`.